### ISSUE TITLE
Development-Process.md / issue 

### ISSUE THREAD

**Issue by as2756** [nasa/fprime#55]
Hello, 
I am trying to set up a port using the directions found at: https://github.com/fprime-community/fprime-course-materials/blob/master/Flight_Software_Workshop-June_2019/Development-Process.md
However, I am having difficulties once I arrive to step 7 (in the port directory: make to autocode the port). 

I am getting an error that "status" was not declared in the scope (screenshots of the error are attached below). I have talked to a couple of my  teammates and they have had the same issue. Does anyone have any ideas what I should do to fix the problem? Thank you!

![VirtualBox_AnjaCisLunar_24_06_2019_19_41_20](https://user-images.githubusercontent.com/42918559/60059096-1bbd0780-96b9-11e9-9151-ad783305da20.png)
![VirtualBox_AnjaCisLunar_24_06_2019_19_41_36](https://user-images.githubusercontent.com/42918559/60059098-1fe92500-96b9-11e9-80c6-7ab7a49d4a39.png)

**Comment by aaronbuchwald** [comment]
Hello, I saw this issue and wasn't sure what to make of it. The error seems to be coming from code that was autogenerated with the word status when it maybe should have written static as in the suggested alternative. Any idea if this is an issue with the autocoders or if it was caused by some error in how the component was written? Would it be helpful to share the code for the component that caused this error?
